src/main/java/codeu/model/store/basic/ActivityStore.java

```diff
@@ -113,39 +113,44 @@ public void addActivity(Activity activity) {
     }
 
     public void addActivity(User user) {
-        String[] displayStringParameters = {user.getCreationTime().toString(),
-                                            user.getName()}; 
+        List<String> displayStringParameters = new ArrayList<>();
+        displayStringParameters.add(user.getCreationTime().toString());
+        displayStringParameters.add(user.getName());        
         Activity activity = new Activity(Activity.ActivityType.UserRegistered, 
                                          user.getId(),
                                          user.getCreationTime(),
                                          displayStringParameters);
+
         activities.add(activity);
         persistentStorageAgent.writeThrough(activity);
     }
 
     public void addActivity(Message message) {
-        String[] displayStringParameters = {message.getCreationTime().toString(),
-                                            UserStore.getInstance()
-                                                .getUser(message.getAuthorId())
-                                                .getName(),
-                                            ConversationStore.getInstance()
-                                                .getConversationWithUUID(message
-                                                .getConversationId()).getTitle(),
-                                            message.getContent()}; 
+        List<String> displayStringParameters = new ArrayList<>();
```

Comment:
When Ricardo submits his pull request we can use Lists.newArrayList() for this instead and create all of this in one line!